tests/ceph_volume/test_ceph_volume_lvm_cli_validations.py

```diff
@@ -0,0 +1,42 @@
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Passing incorrect arguements to ceph-volume lvm
+    prepare, activate and create command should not
+    cause crashes. User understandable error/warning shuld be
+    displayed
+    """
+    # Get configs
+    configs = kw.get("config")
+
+    # Get specs from config
+    data = configs.get("device_path")
+    osd_id = configs.get("osd_id")
+    osd_fsid = configs.get("osd_fsid")
+
+    # Get the installer node
+    node = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Run the command lvm prepare
+    out = CephAdm(node).ceph_volume.lvm.prepare(data)[1]
+    if "lsblk: /dev/vdz: not a block device" in out:
+        log.info("lvm prepare failed as expected")
+
+    # Run the command lvm activate
+    out = CephAdm(node).ceph_volume.lvm.activate(osd_id, osd_fsid)[1]
+    if (
+        "RuntimeError: could not find osd.70 with osd_fsid 299b7964-4753-4711-a449-9b79d70test"
```

Comment:
instead of constant err msg, create a string using passed values e.g
```
err = f"RuntimeError: could not find {osd_id} with osd_fsid {osd_fsid}"
```

Reply:
Done

Reply:
Done